recipes/pybind11_json/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(pybind11_json REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} pybind11_json::pybind11_json)  # FIXME: target should be pybind11_json
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 11)
```

Comment:
```suggestion
add_executable(test_package test_package.cpp)
target_link_libraries(test_package pybind11_json::pybind11_json)  # FIXME: target should be pybind11_json
```

Reply:
gcc 4.9 needs this. It has an older c++ standard by default.

Reply:
hence https://github.com/conan-io/conan-center-index/pull/6462#discussion_r675530015

Reply:
Yes.

Note that all cmake scripts in test packages currently do `set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 11)` instead of using a global `CMAKE_CXX_STANDARD`.

Reply:
not really, its used both in a ~ 60:40 ratio

Reply:
Oh, then it's 59:41 now :smile: